### PR TITLE
Samples readme updated to include features description

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/graph-channel-lifecycle/nodejs/Readme.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ urlFragment: officedev-microsoft-teams-samples-graph-channel-lifecycle-nodejs
 
 This sample show the [Channel](https://docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-1.0) life cycle which includes create, update delete a channel.
 
+## Included Features
+* Tabs
+* Graph API
```
Comment (Pawan-MSFT) [inline samples/graph-channel-lifecycle/nodejs/Readme.md:22]: can we merge these into single heading.

File: samples/graph-pinned-messages/csharp/README.md [hunk 2]
```diff
@@ -15,9 +15,15 @@ urlFragment: officedev-microsoft-teams-samples-graph-pinned-messages-csharp
 
 # This is a sample application which demonstrates how to pin messages in chat using Graph api.
 
-This is an sample application which displays all the pinned messages in group chat. It also demonstrates how to pin new message in the chat.\
+This is an sample application which displays all the pinned messages in group chat. It also demonstrates how to pin new message in the chat.
 
-- **Interaction with app**
+## Included Features
+* Teams SSO (Tabs)
```
Comment (Pawan-MSFT) [inline samples/graph-pinned-messages/csharp/README.md:21]: make consistent.

**Review by Pawan-MSFT — COMMENTED** [review]
Please fix comments

**Review by Pawan-MSFT — APPROVED** [review]
looks correct approving.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/4590222204

**Pull request merged**